### PR TITLE
merging changes from private to public for network

### PR DESCRIPTION
**Pull request by amarzavery**

### CODE REVIEW

**Comment by amarzavery** [comment]
@sergey-shandar please review

**Comment by amarzavery** [comment]
@deepakrajendranmsft please review.

**Comment by amarzavery** [comment]
`Microsoft.Network/2017-06-01/expressRouteConnector.json` this swagger is only present in 2017-06-01 api-version. However, we need this also in 2017-08-01 and 2017-09-01 api-version folders as well. Networking moves together as 1 unit due to models interlinking between different resource types. 
As of today we always publish sdks from the latest api-version. Hence I believe, we should have `expressRouteConnector.json` with it's examples in 2017-08-01 and 2017-09-01 api-versions as well. 
However, I am not sure if the service is deployed with that api-version in all the regions.
@deepakajendranmsft and @dedhar please confirm.

/cc @lmazuel

File: specification/network/resource-manager/readme.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ input-file:
 - Microsoft.Network/2017-06-01/checkDnsAvailability.json
 - Microsoft.Network/2017-06-01/endpointService.json
 - Microsoft.Network/2017-06-01/expressRouteCircuit.json
+- Microsoft.Network/2017-06-01/expressRouteConnector.json
 - Microsoft.Network/2017-06-01/loadBalancer.json
```
Comment (sergey-shandar) [inline specification/network/resource-manager/readme.md]: As discussed before, `expressRouteConnector.json` is missing in `2017-08-01` and `2017-09-01`. We need to confirm with Network team.

**Comment by amarzavery** [comment]
@anuchandy - Please review.

**Review by DeepakRajendranMsft — APPROVED** [review]

**Comment by DeepakRajendranMsft** [comment]
@EvgenyAgafonchikov  please review this.

**Comment by sergey-shandar** [comment]
@amarzavery no changes in `readme.md`?

**Comment by amarzavery** [comment]
@sergey-shandar no changes in readme.md. Public repo already has 2017-09-01 and 2017-08-01 api-version tags and the list of files for an api-version has not changed.

**Pull request merged**